README.md

```diff
@@ -0,0 +1,142 @@
+# Pactum
+## The HTTP-API specification sketchbook for pythonistas
+
+[![Circle CI](https://circleci.com/gh/olist/pactum.svg?style=svg)](https://circleci.com/gh/olist/pactum)
+
+```shell
+pip install pactum
+```
+
+With `Pactum` you can **specify** HTTP-API using pure python.
+
+Pactum is easy to use, easy to extend and easy to contribute:
+
+### Easy to use
+
+The only requirement to start writing an API specification with `pactum`
+is `pactum` package itself and some knowledge of python.
+
+```python
+import pactum
+
+class MyAPI(API):
+    name = 'My API'
+    versions = [...]
+```
+
+### Easy to extend
+
+Using the [visitor pattern](http://wiki.c2.com/?VisitorPattern) you can create
+exporters and extensions for any format or service you want.
+
+Take a look at [pactum/exporters/openapi.py](pactum/exporters/openapi.py).
+
+### Architecture
+
+Always keep this diagram in mind when defining your APIs.
+
+![Architecture Diagram](Diagram.svg)
+
+
+### Tutorial
+
+Create a file called specs.py and start defining your API.
+
+You can define a `Resource` object for your API.
+
+```python
+from pactum import Action, API, Resource, Response, Version
+from pactum import fields, verbs
+
+class MyResource(Resource):
```

Comment:
Is probably more work, but the tutorial could be made defining a simple API with actual resources?

Reply:
Do you have any suggestion? We have an example using "orders" as resource.

Reply:
Looks like a good example